.changeset/wise-laws-retire.md

```diff
@@ -0,0 +1,5 @@
+---
+"@asyncapi/multi-parser": minor
```

Comment:
Remember we also need to release the `@asyncapi/parser` package as previous release failed.

BTW, I'm really concerned about the scenario where releasing a new version of the `@asyncapi/parser` and the fact the `@asyncapi/multi-parser` package will need to be released as well but with an up-to-date `@asyncapi/parser` dependency. 

You mentioned:

> yes that would mean we dont need to do anything else to sync the parser with multiparser it will be automatically synced
> Source: https://github.com/asyncapi/parser-js/pull/1036#issuecomment-2225374094

Because the release process should need to follow an strict order in this case afaik:

1. First, release the new `@asyncapi/parser` version, so the package is out with a new tag.
2. To update the dependency in the `package.json` in the `@asyncapi/multi-parser`

How that will be done automatically?

Reply:
in the ci the new tag will be generated for the @asyncapi/parser and also for the @asyncapi/multiparser

Reply:
I undesrtand. But to what version of the `@asyncapi/parser` will the `@asyncapi/multi-parser` package point to ?

Reply:
ok let me clearfy with example. 

current version of @asyncapi/parser is 3.1.0
now as the multiparser is using the @asyncapi/parser like this 
    "parserapiv3": "file:../parser"

so the it will point to @asyncpi/parser 3.1.0

now let suppose we make some chagnes in @asyncapi/parser though we havent pushed the code and released it yet it will point to latest code. only. here we dont need to think about version for parser dependency in multiparser.

Reply:
Ok, fair enough. Thanks for clarifying!

Reply:
Still this is a valid point:

> Remember we also need to release the @asyncapi/parser package as previous release failed.

Reply:
this is also done automatically by the changeset action it will check the version that is released in the npm and compare it with the latest version which we have if it is not released it will push the release.

Reply:
Ok, I didn't know about that, I thought it was only based on changeset file. Thanks then.

Reply:
It seems it didn't happened, isnt? https://github.com/asyncapi/parser-js/releases

Reply:
it happened for multiparser correctly just not happen for parser because the previous pr should remain opened only.